Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'rails', '~> 4.2.7'
+gem 'rails', '>= 5.0.0.rc2', '< 5.1'
```

Comment:
Latest version of Rails is `5.0.0.1` :smile_cat: 

You can check all the gem versions on https://rubygems.org